evennia/objects/manager.py

```diff
@@ -502,6 +502,10 @@ def copy_object(self, original_object, new_key=None,
         for script in original_object.scripts.all():
             ScriptDB.objects.copy_script(script, new_obj=new_object)
 
+        # copy over all tags, if any
+        for tag in original_object.tags.get():
```

Comment:
Now that I can comment correctly, I should note that this will be a list of tag names (`key`). That is, if the tags to copy have `category` or `data` fields, those will be lost in the copy. There is also a chance your loop fails if there is no- or one tag found. I think the way to get around this would be to get the `Tag` instances themselves instead: 

```python
for tag in original_object.tags.get(return_tagobj=True, return_list=True):
    new_object.tags.add(tag=tag.key, category=tag.category, data=tag.data)
```

Reply:
Also note that the test suite fails with this change, that needs to be fixed before this can merge.